### PR TITLE
Added missing options used by prettyMs function.

### PR DESCRIPTION
**Pull request by silh**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/sindresorhus/pretty-ms
- [X] Increase the version number in the header if appropriate.

### CODE REVIEW

File: types/pretty-ms/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
-// Type definitions for pretty-ms 3.2
+// Type definitions for pretty-ms 4.0
```
Comment (BendingBender) [inline types/pretty-ms/index.d.ts]: Please follow the instructions in the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version) to introduce a major version update.

File: types/pretty-ms/index.d.ts [hunk 2]
```diff
@@ -14,6 +15,9 @@ declare namespace prettyMs {
         msDecimalDigits?: number;
```
Comment (BendingBender) [inline types/pretty-ms/index.d.ts]: [secDecimalDigits](https://github.com/sindresorhus/pretty-ms#secdecimaldigits) option seems to be missing here.

**Review by BendingBender — CHANGES_REQUESTED** [review]

File: types/pretty-ms/index.d.ts (hunk 2 above)
Comment (silh) [reply types/pretty-ms/index.d.ts]: This option is one line above the one you have commented on.

File: types/pretty-ms/index.d.ts (hunk 1 above)
Comment (silh) [reply types/pretty-ms/index.d.ts]: Fixed.

Comment (BendingBender) [reply types/pretty-ms/index.d.ts]: Thank you!

File: types/pretty-ms/index.d.ts (hunk 2 above)
Comment (BendingBender) [reply types/pretty-ms/index.d.ts]: Well, seems like I'm blind.

**Review by BendingBender — APPROVED** [review]

**Review by BendingBender — APPROVED** [review]

**Pull request merged**